test/http2/closing.js

```diff
@@ -99,3 +101,21 @@ t.test('http/2 request while fastify closing - return503OnClosing: false', t =>
     t.end()
   })
 })
+
+// Skipped because there is likely a bug on Node 8.
+t.test('http/2 closes successfully with async await', { skip: semver.lt(process.versions.node, '10.15.0') }, async t => {
```

Comment:
Could we add a test also that check `session.on('timeout', () => { t.pass('timeout out reached') });` ?

Reply:
done